docs/tools/services/relayer-developer.md

```diff
@@ -11,154 +11,187 @@ This feature is currently in private beta. If you are interested, please [contac
+### 2. Register Existing Universal Profiles
 
-</details>
+You can register existing Universal Profiles with the Relayer to enable gasless transactions for your users.
 
-<details>
-  <summary>OPTION 2: salt & postDeploymentCallData</summary>
+```javascript
+async function registerUniversalProfile() {
+  const apiKey = 'your-api-key';
+  const url = 'https://relayer-api.testnet.lukso.network/api/users';
 
-**Why is it useful to deploy UP to pass salt and postDeploymentCallData?**
+  const data = {
+    universalProfileAddress: '0x1234567890123456789012345678901234567890',
+    // userId: 'optional-user-id-for-existing-users',
+  };
 
-To be able to deploy a UP on the same address across different chains.
-
-**How to generate each parameter?**
-
-```javascript title="salt: A 32 bytes salt used to compute the deployment address of the contract."
-const salt = '0x' + crypto.randomBytes(32).toString('hex');
-```
-
-```javascript title="postDeploymentCallData: Calldata which will be executed on the ERC725Account contract after deployment. Should contain the encoded setDataBatch transaction to set the initial permissions and LSP3 Profile Data"
-
-generatePostDeploymentCallData(
-    lsp6Controllers: string[],
-    lsp3Profile?: string,
-  ) {
-    const permissionData: {
-      keyName: string;
-      dynamicKeyParts?: string;
-      value: string[] | string;
-    }[] = [
-      {
-        keyName: 'AddressPermissions[]',
-        value: lsp6Controllers,
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'X-API-KEY': apiKey,
+        'Content-Type': 'application/json',
       },
-    ];
-
-    for (const controller of lsp6Controllers) {
-      permissionData.push({
-        keyName: 'AddressPermissions:Permissions:<address>',
-        dynamicKeyParts: controller,
-        value: ERC725.encodePermissions(DEFAULT_CONTROLLER_PERMISSIONS),
-      });
-    }
-
-    const erc725js = new ERC725(LSP6Schema);
-    const { keys, values } = erc725js.encodeData(permissionData);
-
-    if (lsp3Profile) {
-      keys.push(ERC725YDataKeys.LSP3.LSP3Profile);
-      values.push(lsp3Profile);
-    }
-
-    const postDeploymentCallData = ethers.utils.defaultAbiCoder.encode(
-      ['bytes32[]', 'bytes[]'],
-      [keys, values],
-    );
+      body: JSON.stringify(data),
+    });
 
-    return postDeploymentCallData;
+    const result = await response.json();
+    console.log('Universal Profile registered:', result);
+  } catch (error) {
+    console.error('Error registering Universal Profile:', error.message);
   }
-```
+}
 
-</details>
+registerUniversalProfile();
+```
 
-Please refer to the [Swagger](https://relayer-api.testnet.lukso.network/docs#/External%20Api%20Endpoints/UniversalProfileController_deployUniversalProfile) docs for more information.
+## Integration Guide
 
-### Register Users and Universal Profiles
+1. Obtain API credentials by filling out the access request form.
```

Comment:
Need a link to the form for developers.

Also this form link should be accessible and visible from the top.

```suggestion
1. Obtain API credentials by filling out the access [request form](https:///link-to-form.com).
```